panel/tests/test_template.py

```diff
@@ -18,16 +19,16 @@
 from panel.layout import Row
 from panel.pane import HoloViews, Markdown
 from panel.template import (
-    BootstrapTemplate, GoldenTemplate, MaterialTemplate, ReactTemplate,
+    BootstrapTemplate, FastListTemplate, FastGridTemplate, GoldenTemplate, MaterialTemplate, ReactTemplate,
```

Comment:
Not relevant to this PR. But I discovered these templates where not tested in same way as the other templates.